mivs/models.py

```diff
@@ -318,7 +318,8 @@ def has_video_issues(self):
 
     @property
     def has_game_issues(self):
-        return self.game_status in c.PROBLEM_STATUSES
+        if self.game_status != c.COULD_NOT_PLAY:
```

Comment:
c.COULD_NOT_PLAY represents a game that specifically works... so even though it's a 'problem' status (admins should get notified), it doesn't mean the game has an issue.